react/AppTile/index.jsx

```diff
@@ -52,14 +55,30 @@ export const AppTile = ({
     ? showStatus.indexOf(statusLabel) > -1 && statusLabel
     : showStatus && statusLabel
   IconComponent = IconComponent || AppIcon
+  const isInMaintenance = maintenanceDisplay && statusLabel === 'maintenance'
+
   return (
-    <Tile tag="button" type="button" onClick={onClick}>
+    <Tile
+      tag="button"
+      type="button"
+      onClick={onClick}
+      className={
+        isInMaintenance ? styles['AppTile-container-maintenance'] : undefined
+      }
+    >
       <TileIcon>
         <IconComponent
           app={app}
           className={styles['AppTile-icon']}
           {...getAppIconProps()}
         />
+        {isInMaintenance && (
```

Comment:
Also not a big fan of having severals const for the same thing. We should avoid to rename things. 

This is not `isInMaintenance` then we display... This is `isInMaintenance && displaySpecificMaintenanceStyle` . We should not mix the two concept